src/dmd/globals.h

```diff
@@ -89,6 +89,14 @@ enum CppStdRevision
     CppStdRevisionCpp17 = 201703
 };
 
+/// Configuration for the C++ header generator
+enum CxxHeaderMode
```

Comment:
Please use `enum class`. Is the mixed casing of the members intentional?

Reply:
> Please use `enum class`.

Fixed

> Is the mixed casing of the members intentional?

No, just a typo.